commands/add.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs')
+let tasks = JSON.parse(fs.readFileSync('./tasks.json', 'utf8'))
+
+const add = (newTask) => {
+  let toDo = {}
+  toDo.task = newTask
+  toDo.id = tasks.length + 1
+  tasks.push(toDo)
+  console.log(`Created task ${toDo.id}.`)
+  return JSON.stringify(tasks)
```

Comment:
instead of returning you should put 
```js
fs.writeFileSync('./tasks.json', JSON.stringify(tasks))
``` 
at the bottom of the `add` function and then put 
```js
add(process.argv[3])
``` 
at the bottom of this file.